### DISCUSSION TITLE
2022-2032 Planner PDFs

### DISCUSSION THREAD

**Discussion by thedevscott** [kudrykv/latex-yearly-planner#57] [General]
All,

Here are the pre-generated planners for 2022-2032 for those who are just trying to get a planner for their device.

Best Regards,
thedevscott

### Dotted
- [2022-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666740/2022-Planner.pdf)
- [2023-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666741/2023-Planner.pdf)
- [2024-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666742/2024-Planner.pdf)
- [2025-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666743/2025-Planer.pdf)
- [2026-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666748/2026-Planner.pdf)
- [2027-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666751/2027-Planner.pdf)
- [2028-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666752/2028-Planner.pdf)
- [2029-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666755/2029-Planner.pdf)
- [2030-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666759/2030-Planner.pdf)
- [2031-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666764/2031-Planner.pdf)
- [2032-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666765/2032-Planner.pdf)

### Lined

- [2032-Planner-Lined.pdf](https://github.com/user-attachments/files/24601797/2032-Planner-Lined.pdf)
- [2031-Planner-Lined.pdf](https://github.com/user-attachments/files/24601798/2031-Planner-Lined.pdf)
- [2030-Planner-Lined.pdf](https://github.com/user-attachments/files/24601799/2030-Planner-Lined.pdf)
- [2029-Planner-Lined.pdf](https://github.com/user-attachments/files/24601801/2029-Planner-Lined.pdf)
- [2028-Planner-Lined.pdf](https://github.com/user-attachments/files/24601802/2028-Planner-Lined.pdf)
- [2027-Planner-Lined.pdf](https://github.com/user-attachments/files/24601803/2027-Planner-Lined.pdf)
- [2026-Planner-Lined.pdf](https://github.com/user-attachments/files/24601804/2026-Planner-Lined.pdf)

**Comment by Puddinhead916** [comment]
I downloaded the 2022 Planner and the 2023 Planner to Boox Note Air 2 and SN A5X. I'm having problems on SN whenever I finger-touch anything the planner jumps to a previous date and touching again it jumps to a different date. I set the Air Note 2 to the Mode 6 touch and that eliminated the problem. Any suggestions for the SN?

**Reply by kudrykv** [reply]
Hi @Puddinhead916!
Please clarify:
1. Do you touch of what supposed to be non-link, and it randomly redirects somewhere?
2. When you touch the link, for example, Q4 on the annual page, do you get to Q4 or somewhere else?
3. For example on quarterly pages, when you touch the month name of the mini-calendar, do you land on the corresponding month?

**Comment by Puddinhead916** [comment]
Do you touch of what supposed to be non-link, and it randomly redirects somewhere?
I am touching blank spaces such as 2 PM in the Schedule column and it jumps back one day. Touching the date located over the Schedule column also moves back one day.

When you touch the link, for example, Q4 on the annual page, do you get to Q4 or somewhere else?
Touching the Q4 or the months 'tabs' takes me to those specific areas as intended.
Touching the blank space under Top priorities, for example moves the date back a day.

For example on quarterly pages, when you touch the month name of the mini-calendar, do you land on the corresponding month
Yes, all of that seems to work correctly. Again, when I touch the blank spaces after going to the quarterly pages and then the months in the mini-calendar the day moves back a full day.
________________________________

**Reply by kudrykv** [reply]
I think I know what the problem might be.

Please check what option you have enabled in Supernote to flip pages: whether it's set to one finger. With one finger enabled, pages flip from a touch.

![CleanShot 2022-10-08 at 11 43 03](https://user-images.githubusercontent.com/12697803/194698626-97cda3fa-e853-45e6-a426-7d3d48c64a1d.png)

**Comment by Puddinhead916** [comment]
It’s always set to disable touch to flip. I use the page arrows at the bottom of the page.

Bruce Bailey

On Oct 8, 2022, at 3:43 AM, Vitaliy Kudryk ***@***.***> wrote:

﻿

I think I know what the problem might be.

Please check what option you have enabled in Supernote to flip pages: whether it's set to one finger. With one finger enabled, pages flip from a touch.

[CleanShot 2022-10-08 at 11 43 03]<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F12697803%2F194698626-97cda3fa-e853-45e6-a426-7d3d48c64a1d.png&data=05%7C01%7C%7Cb5a93b28f0254f25943b08daa9093881%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638008154314013949%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=NgFrqvZqOrUrvswPxKpNse5Dl6nsl6AeCINUtNfhViQ%3D&reserved=0>

**Reply by kudrykv** [reply]
> I am touching blank spaces such as 2 PM in the Schedule column and it jumps back one day. Touching the date located over the Schedule column also moves back one day.

That sound like a page flipping. Are you sure you're not somehow triggering a page flip? If you try to click those areas on the computer, nothing happens. That means the issue isn't with the PDF itself.

**Comment by Puddinhead916** [comment]
I’ll check it to be sure.

Bruce Bailey

On Oct 8, 2022, at 8:32 AM, Vitaliy Kudryk ***@***.***> wrote:

﻿

I am touching blank spaces such as 2 PM in the Schedule column and it jumps back one day. Touching the date located over the Schedule column also moves back one day.

That sound like a page flipping. Are you sure you're not somehow triggering a page flip? If you try to click those areas on the computer, nothing happens. That means the issue isn't with the PDF itself.

**Comment by Naxeusyll** [comment]
Hey there! I absolutely love these planners and this format is almost exactly what I'm using now. However, it would be great to have these lined instead of dotted. My LaTeX is absolutely poor -- can I convince you to provide these calendars provided here with lines instead of dots? Thanks so much!

**Reply by kudrykv** [reply]
Hi — yeah, I'll look into it in a couple of days

**Reply by Naxeusyll** [reply]
> Hi — yeah, I'll look into it in a couple of days

I really appreciate that!

**Reply by steelcage** [reply]
I second this request. I am using a Supernote A6X and this one works very well but I just wish it had lines insted of dots.
Thanks,

**Reply by Naxeusyll** [reply]
> Hi — yeah, I'll look into it in a couple of days

Thanks! Just following up on this again for myself and @steelcage.

**Reply by kudrykv** [reply]
Yeah, I'm sorry about that. My offline got in the way, and this got deprioritized real quick.
Honestly, I don't know if I could jump on this. My mental resources are depleted. I can't promise that I'll manage to generate one for you. I can't hop on it if you don't hear from me until ~20 December. I'm sorry.

**Reply by Naxeusyll** [reply]
> Yeah, I'm sorry about that. My offline got in the way, and this got deprioritized real quick. Honestly, I don't know if I could jump on this. My mental resources are depleted. I can't promise that I'll manage to generate one for you. I can't hop on it if you don't hear from me until ~20 December. I'm sorry.

Hey there! I'm sorry to hear you feel/felt so depleted at the end of the year -- I think we all can empathize! I'm coming back in again to see if there is any chance you might be able to complete that request now. I would be happy to throw in 15 bucks for your pain. I tried to learn enough latex to do it myself but I totally failed! No pressure though: be well, it's what we all want from you most!

**Reply by kudrykv** [reply]
You should be able to edit the config and recompile it
Please check 
[sn_a5x.breadcrumb.lined.default.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/10335817/sn_a5x.breadcrumb.lined.default.2023.pdf)

**Reply by Boshimanu** [reply]
Hi man, how do you pass from dotted to lined, I get default as dotted... what do you have to modify? Thank you!

**Reply by wanhoff** [reply]
I think this one:

File: cfg/base.yaml
dotted: true

try setting it to false instead of true.

**Reply by SwaroopKM07** [reply]
@kudrykv  Can you please provide 2024 and 2025 calender with lines? It would be a great help.

**Comment by JohnCEEllis** [comment]
Thanks, great download.

I do write my own templates and add them to the RM2 template list, but each update wipes this out.

So many people selling these templates, when so many are free.

**Comment by Puddinhead916** [comment]
On the other hand, I hope you will keep the lines or add the grid option!

Bruce Bailey

On Nov 18, 2022, at 1:26 PM, steelcage ***@***.***> wrote:

﻿

I second this request. I am using a Supernote A6X and this one works very well but I just wish it had lines insted of dots.
Thanks,

**Comment by poloman172** [comment]
Just wanted to say how amazing these PDFs are - I found them by chance and have imported into Noteability and Wow! Game changed!

Thanks for all the hard work that has gone into them - I'm almost tempted to try Latex to get those ToDoLists back. My daily process is now under control and I'm fully organized - a great way to start 2023.

**Reply by kudrykv** [reply]
It's possible to find ones with ToDos in older versions. They didn't differ much and they also can compile planners for any specified year. There's also work-in-progress on the refactor branch, but it takes a little while

**Reply by poloman172** [reply]
No worries - I'm using the non-ToDoList ones (the 10 year set that someone kindly published) with Notes pages pretty effectively. I won't switch out from my populated noteability copy this year so gives me near enough a year to figure out if I need ToDoLists enough to learn Latex. 
Thanks again for all the work - I've even got my wife to switch away from her paper Filofax!

**Comment by RCS507** [comment]
Thank you very much for these.  You are awesome.  I loaded these on a Kindle Scribe that I just bought to test and they take the device into a whole new category.  Love the interaction and navigation.  Created an account with Github just to thank you!

**Reply by moslem-hadi** [reply]
How did you transfer the file to kindle? When I send it to kindle scribe, the pen doesnt work. There is no pen toolbar for it.

**Reply by nanshee** [reply]
@moslem-hadi  - use the SEND TO KINDLE function and you'll be able to edit/write on it. good luck~  https://www.amazon.com/sendtokindle

**Comment by hcdoro** [comment]
Hi All,

I really love the sample and value of the long digital planner (2022 - 2032), but it was not fit well to my boox nova air c screen dimension, 7.8" , 136.5mm × 194mm × 6.3mm. 
Can anyone help to convert into those size? Please...

**Comment by justarmadillo** [comment]
THANK YOU !

**Comment by kwgp** [comment]
This is amazing; thoughtful layout, design, and functionality (hyperlinks are well ordered and work as expected). Just starting to test out moving from paper to digital planners and this is incredibly helpful. Really appreciate you sharing it, thank you! 🙌


_As a side-note, if anyone is having trouble with "hyperlinks not working" or "pages jumping around" do check that you're in 'read mode' and look at support threads for your specific app/software. I tested this in Goodnotes and Samsung Notes and it works perfectly in both. Mentioning since there were some comments above indicating an issue with some functionality that I have to assume are user-error, or perhaps an in-app setting of sorts (does not appear to be a design issue)._

**Comment by Remi-Sho** [comment]
I love this planner. Thank you so much for sharing it for free!!

I was wondering if you could create a version with am/pm instead of 24 hour clock times? I'd also love it if the dots in the notes areas were a lighter gray. If you decide to make this version let me know and I'll send you some thanks!!

**Comment by pixiedelmuerte** [comment]
Thanks for these! They are the *precise* layout I've used in analog planners for ages.

**Comment by EZ707** [comment]
it would be great to have separate  todo and notes

**Comment by PolianaMihaela** [comment]
> All,
> 
> Here are the pre-generated planners for 2022-2032 for those who are just trying to get a planner for their device.
> 
> Best Regards, thedevscott
> 
> * [2022-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666740/2022-Planner.pdf)
> * [2023-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666741/2023-Planner.pdf)
> * [2024-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666742/2024-Planner.pdf)
> * [2025-Planer.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666743/2025-Planer.pdf)
> * [2026-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666748/2026-Planner.pdf)
> * [2027-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666751/2027-Planner.pdf)
> * [2028-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666752/2028-Planner.pdf)
> * [2029-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666755/2029-Planner.pdf)
> * [2030-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666759/2030-Planner.pdf)
> * [2031-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666764/2031-Planner.pdf)
> * [2032-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666765/2032-Planner.pdf)

Hi, I got a weekly planner from you 2 yrs ago, lined weekly with hours included, is that still available for 2025? Thanks!!

**Reply by thedevscott** [reply]
Hello Poliana,
The calendars/planners are still available. I just downloaded the 2025 version by clicking:
- [2025-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666743/2025-Planer.pdf)
All the best in planning your new year.

**Comment by aladjem** [comment]
This is amazing work!

any chance you’d make versions with Sunday as the start of the week rather than Monday?

**Reply by asherov** [reply]
Oh wow, that would be huge. Fingers crossed.

**Reply by poloman172** [reply]
If you use Claude Code, you can get it to use Sunday as the start of week.
I intended to learn how to compile and configure the source code, but Claude does everything - even installing the tools needed. 
See the attachment for 2026 with Sundays as the start of the week:

[planner.2026.kindle.breadcrumb.lines.sunday.pdf](https://github.com/user-attachments/files/23786512/planner.2026.kindle.breadcrumb.lines.sunday.pdf)

**Comment by asherov** [comment]
This is absolutely amazing – thank you so much!

**Comment by brdavidsonwa-boop** [comment]
These are fantastic planners and have been using since 2023. Are there lined versions available for 2027-2032 like there are for the dotted version?

**Reply by poloman172** [reply]
Claude code takes the win here - I set it up using their desktop app and told it what i wanted - it installed everything and translates my random instruction into the output. It commented on how well documented the source code was too!

I've attached a file for 2026 with lines as an example....
[planner.2026.kindle.breadcrumb.lines.pdf](https://github.com/user-attachments/files/23786531/planner.2026.kindle.breadcrumb.lines.pdf)

**Reply by thedevscott** [reply]
Hi @brdavidsonwa-boop, lined versions have been uploaded. See the [first post](https://github.com/kudrykv/latex-yearly-planner/discussions/57#discussion-4429366).

Best Regards,
Scott

**Comment by ryderdavid** [comment]
> All,
> 
> Here are the pre-generated planners for 2022-2032 for those who are just trying to get a planner for their device.
> 
> Best Regards, thedevscott
> 
>     * [2022-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666740/2022-Planner.pdf)
> 
>     * [2023-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666741/2023-Planner.pdf)
> 
>     * [2024-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666742/2024-Planner.pdf)
> 
>     * [2025-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666743/2025-Planer.pdf)
> 
>     * [2026-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666748/2026-Planner.pdf)
> 
>     * [2027-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666751/2027-Planner.pdf)
> 
>     * [2028-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666752/2028-Planner.pdf)
> 
>     * [2029-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666755/2029-Planner.pdf)
> 
>     * [2030-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666759/2030-Planner.pdf)
> 
>     * [2031-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666764/2031-Planner.pdf)
> 
>     * [2032-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666765/2032-Planner.pdf)

Hi there @thedevscott , do you still have the actual template spec for this version you created? I want to rebuild it for myself with some minor changes. Thank you!

**Reply by thedevscott** [reply]
Hi @ryderdavid, apologies for the delay.

Can you give me an example of what you are looking for? I have not touched this project in some years so a concrete example will help with the search.

I assume the template you are looking for is in "cfg/template_breadcrumb.yaml".

The command used to build the pdf is:
<code>
PLANNER_YEAR=2026 PASSES=1 CFG="cfg/base.yaml,cfg/template_breadcrumb.yaml,cfg/sn_a5x.breadcrumb.default.yaml" NAME="sn_a5x.breadcrumb.default" ./single.sh
</code>

The above command should all be on one line.

To switch between dotted and lined you would change the "dotted" flag in "cfg/base.yaml" on line 3 to false.

Best Regards,
Scott

**Comment by PolianaMihaela** [comment]
Do you have the version with lines instead of dots?

On Fri 2 Jan 2026 at 09:24, Ryder Cobean ***@***.***> wrote:

> All,
>
> Here are the pre-generated planners for 2022-2032 for those who are just
> trying to get a planner for their device.
>
> Best Regards, thedevscott
>
> * [2022-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666740/2022-Planner.pdf)
>
> * [2023-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666741/2023-Planner.pdf)
>
> * [2024-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666742/2024-Planner.pdf)
>
> * [2025-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666743/2025-Planer.pdf)
>
> * [2026-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666748/2026-Planner.pdf)
>
> * [2027-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666751/2027-Planner.pdf)
>
> * [2028-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666752/2028-Planner.pdf)
>
> * [2029-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666755/2029-Planner.pdf)
>
> * [2030-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666759/2030-Planner.pdf)
>
> * [2031-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666764/2031-Planner.pdf)
>
> * [2032-Planner.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9666765/2032-Planner.pdf)
>
> Hi there @thedevscott <https://github.com/thedevscott> , do you still
> have the actual template spec for this version you created? I want to
> rebuild it for myself with some minor changes. Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kudrykv/latex-yearly-planner/discussions/57#discussioncomment-15389338>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BOEFCUZUTUQZDIDJX4JFVLD4EYMI7AVCNFSM6AAAAACKGUU6IWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTKMZYHEZTGOA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> .com>
>
-- 
Kind regards,

Mihaela Poliana Aniculaesei

Counsellor/ Psychotherapist
IACP Accreditation
085 168 8381

**Reply by ryderdavid** [reply]
Dots!

**Reply by thedevscott** [reply]
Hi @PolianaMihaela, lined versions have been uploaded to the [first post](https://github.com/kudrykv/latex-yearly-planner/discussions/57#discussion-4429366).

Best Regards,
Scott

**Comment by PolianaMihaela** [comment]
U are a star!!! May I have your buy me a coffee e link? I used it at some
point I can’t find it now

On Tue 13 Jan 2026 at 23:25, Omar Scott ***@***.***> wrote:

> Hi @PolianaMihaela <https://github.com/PolianaMihaela>, lined versions
> have been uploaded to the first post
> <https://github.com/kudrykv/latex-yearly-planner/discussions/57#discussion-4429366>
>
> Best Regards,
> Scott
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kudrykv/latex-yearly-planner/discussions/57#discussioncomment-15490331>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BOEFCU3IDWIQ7D6MU7LQFPL4GV5F3AVCNFSM6AAAAACKGUU6IWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTKNBZGAZTGMI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> .com>
>
-- 
Kind regards,

Mihaela Poliana Aniculaesei

Counsellor/ Psychotherapist
IACP Accreditation
085 168 8381

**Comment by gingnic** [comment]
Thank you very much for your good work and your generosity.
I found your work by chance and I stopped using a planner I already had because yours is very well designed. Thank you again.

I had an idea that I wanted to share with you, if you generate a planner for each of seven regular years beginning by a different day of the week and do the same for seven bissextile years and omit to print the year on the planner, you will be able to share perpetual planners.

Kinds regards
Nicole

**Comment by colojohn** [comment]
Hi! I just downloaded the lined versions of the 2026 and 2027 planners.

Is there a way to sync the planner calendars to my google calendar? That would be terrific because I usually make my appointments on my Android phone.

Thank you,
John Miller

**Reply by kudrykv** [reply]
Hi, it is not supported. PDF is the final document, making the request very tricky.
